Update DB + Migration documentation (#2041)
cisagov/manage.get.gov: docs/developer/migration-troubleshooting.md

2. `./manage.py migrate model_name_here file_name_WITH_create` (run the last data creation migration AND ONLY THAT ONE)
3. `./manage.py migrate --fake model_name_here most_recent_file_name` (fake migrate the last migration in the migration list)
4. `./manage.py load` (rerun fixtures)

### Scenario 9: Inconsistent Migration History
If you see `django.db.migrations.exceptions.InconsistentMigrationHistory` error, or when you run `./manage.py showmigrations` it looks like:

[x] 0056_example_migration
[ ] 0057_other_migration
[x] 0058_some_other_migration

> 1. Go to `database-access.md` to see the commands on how to access a certain table in the database.

Author: Ah great catch to point it to the exact spot, thanks!

2. In this case, we want to remove the migration "history" from the `django_migrations` table

> 3. Once you are in the `cgaws....` table, select the `django_migrations` table with the command `SELECT * FROM djangomigrations;`

Reviewer: ```suggestion
3. Once you are in the `cgaws....` table, select the `django_migrations` table with the command `SELECT * FROM django_migrations;`
```

> 4. Find the id of the "history" you want to delete. In this example, the id would be 58.

Reviewer: ```suggestion
4. Find the id of the "history" you want to delete. This will be the one in the far left column. For this example, let's pretend the id is 101
```

id is not equal to the migration number and don't want anyone to assume they are the same. (The one today was migration 80 but it's id was 118

Author: Ah great catch, thank you!

> 5. Run `DELETE FROM django_migrations WHERE id=58;` where 58 is an example id as seen above.

Reviewer: ```suggestion
5. Run `DELETE FROM django_migrations WHERE id=101;` where 101 is an example id as seen above.
```

> 6. Go to your shell and run `./manage.py showmigrations` to make sure your migrations are now back to the right state

Reviewer: ```suggestion
6. Go to your shell and run `./manage.py showmigrations` to make sure your migrations are now back to the right state. Most likely you will show several unapplied migrations.
```

> 

Reviewer: 7. If you still have unapplied migrations, run `./manage.py migrate`. If an error occurs saying one has already been applied, fake that particular migration and then run the normal `./manage.py migrate` command to then apply those migrations that come after the one that threw the error.
